scripts/batch_preprocess_known_faces.py

```
@@ -0,0 +1,31 @@
+def preprocess_and_save(face_img_path, save_path):
+    img = load_and_correct_orientation(face_img_path)
+    if len(img.shape) == 3:
+        img = cv2.cvtColor(img, cv2.COLOR_BGR2GRAY)
+    img = cv2.resize(img, FACE_SIZE)
+    img = apply_clahe(img)
+    img = cv2.GaussianBlur(img, (3, 3), 0)
+    kernel = np.array([[0, -1, 0], [-1, 5, -1], [0, -1, 0]])
+    img = cv2.filter2D(img, -1, kernel)
+    cv2.imwrite(save_path, img)
+
+def batch_preprocess_known_faces():
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)